rtp_h264_extractor.lua

```diff
@@ -117,15 +117,19 @@ do
         end
         
         local function handle_stap_a(h264_data)
-            offset = 1
+			log("start dump stap nals")
```

Comment:
Do you mean indenting at source code? OK, made them using spaces instead of tabs :)